spec/models/task_spec.rb

```diff
@@ -27,6 +27,58 @@
 
   end
 
+  describe "" do
```

Comment:
It was like a little bit noise putting a title there. I can complete it if you feel uncomfortable about it